### PR TITLE
serve metrics handler with prometheus exporter-toolkit

### PR DESCRIPTION
**Pull request by rlankfo**
This PR serves the metrics handler utilizing `prometheus/exporter-toolkit` instead of `percona/exporter_shared`. This is a start towards fixing https://github.com/percona/mongodb_exporter/issues/319, at least partially, however this does not remove all references to `exporter_shared`. 

Note that TLS Config still needs to be handled by adding an option to pass the tlsConfigPath.

- [ ] Links to other linked pull requests (optional).

---

- [ ] Tests passed.
- [ ] Fix conflicts with target branch.
- [ ] Update jira ticket description if needed.
- [ ] Attach screenshots/console output to confirm new behavior to jira ticket, if applicable.

Once all checks pass and the code is ready for review, please add `pmm-review-exporters` team as the reviewer. That would assign people from the review team automatically. Report any issues on our [Forum](https://forums.percona.com) or [Discord](https://per.co.na/discord).

### CODE REVIEW

**Comment by percona-csalguero** [comment]
Hi, 
Can you remove `go-kit/log` dependency. Since it is used only once, I think it is better to keep 3rd party dependencies at minimum.

Thanks.

File: exporter/exporter.go [hunk 1]
```diff
@@ -21,11 +21,14 @@ import (
 	"context"
 	"fmt"
 	"net/http"
+	"os"
 	"sync"
 
-	"github.com/percona/exporter_shared"
+	"github.com/go-kit/log/level"
```
Comment (percona-csalguero) [inline exporter/exporter.go:27]: Can we remove this dependency?
It is used only once so, I think it is better to keep 3rd party dependencies at minimum.

Thanks.

**Review by percona-csalguero — CHANGES_REQUESTED** [review]

**Comment by percona-csalguero** [comment]
I really appreciate this PR. Using exporter shared was preventing us to implement some other features like SSL support.
Thanks @rlankfo

File: exporter/exporter.go (hunk 1 above)
Comment (rlankfo) [reply exporter/exporter.go]: updated :+1:

Comment (rlankfo) [reply exporter/exporter.go]: Note this will still get pulled in as a transitive dependency of `prometheus/exporter-toolkit`

**Review by percona-csalguero — APPROVED** [review]

**Comment by denisok** [comment]
@JiriCtvrtka please review

File: .gitignore [hunk 2]
```diff
@@ -6,3 +6,4 @@ bin
 build
 dist
 mongodb_exporter
+vendor
```
Comment (JiriCtvrtka) [inline .gitignore:9]: This is not necessary since we dont have vendor folder anymore.

**Review by JiriCtvrtka — APPROVED** [review]
LGTM only one minor comment.

**Comment by denisok** [comment]
@percona-csalguero would test it a little bit more and merge it

**Pull request merged**